packages/zora/src/Zora.ts

```diff
@@ -471,6 +472,12 @@ export const getFees = async (
       publicClient: client,
     })
 
+    // if contract type is 1155, we need to get the latest tokenId
+    if (contractType === '1155' && tokenId == null) {
```

Comment:
if we pass tokenId into `getNextTokenId` it should account for the existing `tokenId` right? Neutral on if we pass it or implementation, important thing is that the simulation and fees are both using the same logic to determine the token to use (both use next if tokenId is undefined)

Reply:
getNextTokenId passes back the existing tokenId if it is defined. 

Felt this was more semantic.

SimulateMint and getFees will both use `getNextTokenId` when no token is available.